firebase-vertexai/src/test/java/com/google/firebase/vertexai/InternalTests.kt

```diff
@@ -24,7 +24,7 @@ import kotlinx.serialization.encodeToString
 import kotlinx.serialization.json.Json
 import org.junit.Test
 
-internal class SchemaTests {
```

Comment:
Since these do test the schema, it makes sense to leave it with the old name

Reply:
Seems to have been an automatic rename... resolved